### PR TITLE
Simple IdentityWallet support in ThreeId class

### PR DESCRIPTION
**Pull request by oed**
Breaking change: KeyValueStore.log is now an async function and not a property

### CODE REVIEW

**Comment by oed** [comment]
Needs a rebase.

**Comment by oed** [comment]
Cant figure out what's up with the tests. The snapshots randomly fail. It works locally, and it works if I ssh into circleci and run them there. Not sure how to tackle yet.

**Comment by oed** [comment]
Looks like there where a few places where `expect(...).rejects.to...´ was missing an `await` before it. Should be fixed now.

**Comment by oed** [comment]
Rebased and ready to be reviewed @zachferland

File: src/3id/index.js [hunk 1]
```diff
@@ -167,7 +208,53 @@ class ThreeId {
...
   }
 
-  logout() {
+  async getPublicKeys (space, uncompressed) {
+    let pubkeys
+    if (this.idWallet) {
+      pubkeys = Object.assign({}, space ? this._pubkeys.spaces[space] : this._pubkeys.main)
+      //console.log(space)
+      //console.log(this._pubkeys)
+      //console.log(pubkeys)
```
Comment (zachferland) [inline src/3id/index.js]: logs needed?

File: package.json [hunk 2]
```diff
@@ -9,7 +9,6 @@
   "scripts": {
     "lint": "./node_modules/.bin/standard --verbose src/**",
     "test": "rm -rf ./tmp && jest --forceExit --coverage --runInBand --testURL=\"http://localhost\"",
-    "test-update": "rm -rf ./tmp ; jest --forceExit --coverage --runInBand --testURL=\"http://localhost\" --updateSnapshot",
```
[the remark this answers is not in the document]
Comment (oed) [reply package.json]: I created it, but the I realized you could just do `npm t path/to/file.js -- -u` to do the same thing

File: src/3id/index.js (hunk 1 above)
Comment (oed) [reply src/3id/index.js]: woops.

**Pull request merged**